Project.toml

```diff
@@ -4,20 +4,22 @@ authors = ["Zachary Sunberg <sunbergzach@gmail.com>"]
 version = "0.2.0"
 
 [deps]
-BasicPOMCP = "d721219e-3fc6-5570-a8ef-e5402f47c49e"
 BeliefUpdaters = "8bb6e9a1-7d73-552c-a44a-e5dc5634aac4"
-FIB = "13b007ba-0ca8-5af2-9adf-bc6a6301e25a"
+D3Trees = "e3df1716-f71e-5df9-9e2d-98e193103c45"
+DiscreteValueIteration = "4b033969-44f6-5439-a48b-c11fa3648068"
+MCTS = "e12ccd36-dcad-5f33-8774-9175229e7b33"
 POMDPModelTools = "08074719-1b2a-587c-a292-00f91cc44415"
 POMDPModels = "355abbd5-f08e-5560-ac9e-8b5f2592a0ca"
 POMDPPolicies = "182e52fb-cfd0-5e46-8c26-fd0667c990f4"
 POMDPSimulators = "e0d0a172-29c6-5d4e-96d0-f262df5d01fd"
+POMDPToolbox = "0729bffe-8e6b-52fa-a3fa-893719b744f4"
```

Comment:
This probably shouldn't be here.

Reply:
I agree, I accidentally left it in after playing with the legacy notebook. I removed it in the next commit.